can_tools/bootstrap_data/covid_units.csv

```diff
@@ -1,5 +1,6 @@
 name
 people
+doses_per_1k
```

Comment:
We're most interested in the raw number of doses -- We can always convert using population on our own. It's possible there will be somewhere that only reports per 1k or per 100k and we can add this in at that stage, but, for now, I'm going to delete this and the corresponding entry in the column renamer.